src/sql/platform/metadata/common/metadataService.ts

```diff
@@ -14,13 +14,13 @@ export const IMetadataService = createDecorator<IMetadataService>(SERVICE_ID);
 export interface IMetadataService {
 	_serviceBrand: undefined;
 
-	getMetadata(connectionUri: string): Thenable<azdata.ProviderMetadata>;
+	getMetadata(connectionUri: string): Thenable<azdata.ProviderMetadata | void>;
```

Comment:
>Thenable [](start = 37, length = 8)

Promise for the ones in this file as well?

Reply:
Others in the other service files too if you wanted to fix them

---
In reply to: [325389256](https://github.com/microsoft/azuredatastudio/pull/7243#discussion_r325389256) [](ancestors = 325389256)